src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/PlacementRequestsController.kt

```diff
@@ -79,7 +79,7 @@ class PlacementRequestsController(
     arrivalDateStart: LocalDate?,
     arrivalDateEnd: LocalDate?,
     requestType: PlacementRequestRequestType?,
-    apAreaId: UUID?,
+    cruManagementAreaId: UUID?,
```

Comment:
@davidatkinsuk looking at this I think this is wrong - from memory the front end isn't changing so no need to change `apAreaId` to `cruManagementAreaId`.  Will wait for you to confirm before reverting this.

Reply:
Correct - we need to retain apAreaId until the front end has moved over to use cruManagementArea instead